draft-ietf-oauth-status-list.md

```diff
@@ -787,6 +787,8 @@ The HTTP response SHOULD use gzip Content-Encoding as defined in {{RFC9110}}.
 
 If caching-related HTTP headers are present in the HTTP response, Relying Parties MUST prioritize the exp and ttl claims within the Status List Token over the HTTP headers for determining caching behavior.
 
+Note that while the examples for Status List Tokens in CWT format in this document are provided in hex encoding, this does not mean that hex encoding is expected for HTTP messages. The expected encoding in an HTTP response containing a Status List Token in CWT format is binary encoding as defined in section 9.2.1 of {{RFC8392}}. This means that the body of an HTTP response contains the raw CBOR bytes.
```

Comment:
Does it make more sense to add this to the paragraph of line 784?

Reply:
I was thinking about that initially as well since it directly fits to the media type, but it kinda made sense to me to not put it in between the normative statements and instead as a note  afterwards.